Remotion/Web/Development.WebTesting/WebDriver/Configuration/Edge/ExtendedEdgeOptions.cs

```diff
@@ -16,19 +16,22 @@
 // 
 using System;
 using JetBrains.Annotations;
+using Microsoft.Edge.SeleniumTools;
 using OpenQA.Selenium.Chrome;
 
 namespace Remotion.Web.Development.WebTesting.WebDriver.Configuration.Edge
 {
   /// <summary>
   /// Extends <see cref="ChromeOptions"/> provided by Selenium with additional data used by the web testing infrastructure.
   /// </summary>
-  public class ExtendedEdgeOptions : ChromeOptions
+#pragma warning disable CS3009

```

Comment:
This baffled me so much that I opened an issue asking them what this is all about:
https://github.com/microsoft/edge-selenium-tools/issues/4

In their source code I couldn't find the attribute to explicitly turn CLS-compliance off, and didn't find anything myself either. Perhaps it's unintended after all, let's see if they answer

Reply:
The CLSCompliantAttribute is opt-in IIRC. Need to check how you specify this with sdk-style project / generated AssebmlyInfo.cs but yes, it certainly looks like they might have forgotten to mark their assembly CLSCompliant.

Reply:
@MichaelKetting @iam-mholle The maintainers responded to my issue and promptly opened a PR to mark the assembly as CLS-compliant https://github.com/microsoft/edge-selenium-tools/pull/10

Reply:
Great, maybe we should wait a couple of days and see how long it takes them to make a release so we can catch this in the current PR?

Reply:
Erm, just read the reply on the other thread. Sounds like this should be done fairly quickly so we should be able to wait with our PR until the update's here.